x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/install_prebuilt_rules_fleet_package.ts

```diff
@@ -73,11 +73,10 @@ export const installPrebuiltRulesFleetPackage = async ({
       installPrebuiltRulesFleetPackage.name,
       async () => {
         const testResponse = await supertest
-          .post(epmRouteService.getBulkInstallPath())
+          .post(epmRouteService.getInstallPath('security_detection_engine', ''))
           .query({ prerelease: true })
           .set('kbn-xsrf', 'true')
           .send({
-            packages: ['security_detection_engine'],
```

Comment:
Hey security folks, we've decided to forbid bulk installations of this package due to its large size causing memory issues during bulk installation requests that include it. Let me know if this approach is acceptable to you. Thanks!

Reply:
> This is correct, we install those two packages in the app with the /fleet/epm/packages/_bulk endpoint. We would need to replace how we install these packages before this is merged.

+1 - I can make this change as part of the PR. Realistically for a bulk install of two packages the performance difference should be entirely negligible between a bulk install or two separate requests.

Reply:
> Hey @kpollich, can you expand on how changing from a bulk install of the prebuilt rules package to a regular install solves the memory issues? I'm not sure I get that bit. We have to install that package anyway, regardless of the installation method, whether it's bulk or something else.

The problem we're seeing with memory usage comes when larger bulk install requests occur, e.g. we see requests in QA serverless projects where the following packages are installed via the bulk API:

```
"security_detection_engine",
"azure",
"apache",
"aws",
"elastic_agent",
"fim",
"fleet_server",
"iis",
"microsoft_sqlserver",
"mysql",
"mysql_enterprise",
"nginx",
"o365",
"oracle",
"osquery_manager",
"postgresql",
"system",
"system",
"ti_abusech",
"ti_util",
```

I was able to reliably trigger an out of memory by making a bulk install request that includes these packages, and the main culprit is the `security_detection_engine` package which has a massive memory footprint. 

Removing the prebuilt rules package from the bulk installation allows it to succeed without memory issues, so that's the intent of this PR. 

Another potential less breaking option would be to sniff out the prebuilt rules package in a bulk request, install it by itself, then batch through the remaining packages as normal in the bulk API. The problem with this approach, at least as it seemed to me initially, is that memory usage remains high even after the installation of the prebuilt rules package is completed. I'm assuming there's some delay in garbage collection or something at play there. Hence, I went with this more far-reaching approach.

Reply:
I'm definitely game for trying the less breaking approach here again, as altering these install paths in the security app is starting to feel like a bit too much scope for this change.

Reply:
I remember some time ago digging into the details of how saved objects import works, and there was the possibility to import saved objects without reading them all into memory by using streams instead. Have you considered adding streamed package installation to Fleet? This might solve the memory issues once and for all, as our package size is ever-growing, and sooner or later there will be even more saved objects and we might again encounter the same memory issues.

Reply:
We do provide a `Readable` stream to the Saved Object importer, but Fleet currently has to read all Kibana assets for a given package into memory first for validation purposes, and to set `references` on the parent `Installation` saved object that captures the overall package installation data. There's no way for us to stream the `references` setting logic we need here, as far as I'm aware, so this would be a heavier lift to alter these Kibana core APIs to allow streaming references. 

You're right that this is the root issue with memory here, and I'll make sure this is on a tech debt backlog as the root cause fix.

Reply:
@kpollich Looks like this PR doesn't affect Security Solution anymore, and everything will be functioning as is without breaking changes?